airflow/dags/airtable_views/california_transit_map_gtfs_service_data_reference_static_gtfs_service_x_self.sql

```diff
@@ -0,0 +1,33 @@
+---
+operator: operators.SqlToWarehouseOperator
+
+dst_table_name: views.airtable_california_transit_map_gtfs_service_data_reference_static_gtfs_service_x_self
+
+description: Self-join mapping table for the GTFS gtfs_service_data table, reference_static_gtfs_service column in the California Transit Airtable base. Each row represents a relationship between two gtfs_service_data records.
+
+fields:
+  gtfs_service_data_id: Internal Airtable ID for a gtfs_service_data record
+  reference_static_gtfs_service_id: Internal Airtable ID for a gtfs_service_data record
+  gtfs_service_data_name: gtfs_service_data record name
+  reference_static_gtfs_service_name: gtfs_service_data record name
+
+tests:
+  check_null:
+    - gtfs_service_data_id
+    - reference_static_gtfs_service_id
+    - gtfs_service_data_name
+    - reference_static_gtfs_service_name
+  check_composite_unique:
+    - gtfs_service_data_id
+    - reference_static_gtfs_service_id
+
+external_dependencies:
+  - airtable_loader: california_transit_gtfs_service_data
+---
+{{
+
+  sql_airtable_mapping(
+    table1 = "gtfs_service_data",table2 = "", col1 = "reference_static_gtfs_service", col2 = ""
```

Comment:
oi, this brings back [memories](https://cal-itp.slack.com/archives/C02JFS4LAMU/p1638909693026300?thread_ts=1638810002.018000&cid=C02JFS4LAMU)

<img width="336" alt="Screen Shot 2022-02-10 at 1 42 34 PM" src="https://user-images.githubusercontent.com/3112493/153501345-449d0a67-cf0a-4bc1-a1aa-147f3cad98b5.png">